frontend/pages/share.js

```diff
@@ -20,13 +20,13 @@ export default function Share({
   const { user } = useContext(UserContext);
   if (!user)
     return (
-      <WideLayout title="Please log in to share a project" hideHeadline={true}>
+      <WideLayout title="Please Log In to Share your Climate Solution" hideHeadline={true}>
```

Comment:
I think if we use capital letters for `Climate Solution` we should also use capital letters for `To` and `Your`. Otherwise it's inconsistent

Reply:
We have talked about it. Either we have to write all words in lower case or we have to follow the english headline capitalization rules. In this case nearly all words are written in upper case except articles, coordinating conjunctions and short prepositions. But I think there are more than one main rule and possible written styles :)